Utilities/Mergers/src/MergerBuilder.cxx

```diff
@@ -67,7 +67,7 @@ framework::DataProcessorSpec MergerBuilder::buildSpec()
 {
   framework::DataProcessorSpec merger;
 
-  merger.name = mergerIdString() + "-" + mName + std::to_string(mLayer) + "l-" + std::to_string(mId);
+  merger.name = mergerIdString() + "-" + mConfig.detectorName + "-" + mName + std::to_string(mLayer) + "l-" + std::to_string(mId);
```

Comment:
you could consider switching the order to comply with the order in QC

Reply:
ok